src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -118,6 +118,7 @@
 import net.minecraftforge.fluids.IFluidBlock;
 import net.minecraftforge.fml.common.eventhandler.Event;
 import net.minecraftforge.fml.relauncher.ReflectionHelper;
+import net.minecraftforge.event.entity.player.CriticalHitEvent;
```

Comment:
well the last time I tried the correct order and it resulted in a merge conflict :/